src/workerd/io/request-tracker.c++

```diff
@@ -28,6 +28,7 @@ RequestTracker::ActiveRequest::~ActiveRequest() {
       }
     }
   }
+  parent = nullptr;
```

Comment:
nit: You don't have to worry about this, the contents of the maybe will be dtor'd as part of destroying the `ActiveRequest`.